app/assets/stylesheets/base/typography.scss

```diff
@@ -0,0 +1,51 @@
+@import './mixins';
+
+@font-face {
+  font-family: OpenDyslexic;
+  @include font-src('OpenDyslexic-Regular.otf');
+  font-style: normal;
+  font-weight: normal;
+}
+@font-face {
+  font-family: OpenDyslexic;
+  @include font-src('OpenDyslexic-Italic.otf');
+  font-style: italic;
+  font-weight: normal;
+}
+@font-face {
+  font-family: OpenDyslexic;
+  @include font-src('OpenDyslexic-Bold.otf');
+  font-weight: bold;
+  font-style: normal;
+}
+@font-face {
+  font-family: OpenDyslexic;
+  @include font-src('OpenDyslexic-BoldItalic.otf');
+  font-weight: bold;
+  font-style: italic;
+}
+
+// Content fonts
+body {
+  --content-font-family: var(--ff-serif);
+
+  &.serif-article-body {
+    --content-font-family: var(--ff-serif);
+  }
```

Comment:
This class name doesn't exist but in spirit of what @benhalpern said re letting admin set their own "default" font I assumed this class might be needed.